frontends/pytorch/test/acap_export/test_export_resnet18.py

```diff
@@ -0,0 +1,28 @@
+# -*- Python -*-
+# This file is licensed under a pytorch-style license
+# See frontends/pytorch/LICENSE for license information.
+
+import torch
+import torch_mlir
+import torchvision.models as models
+
+# TODO: Fix https://github.com/llvm/mlir-npcomp/issues/80
+# XFAIL: *
+# RUN: %PYTHON %s | npcomp-opt | FileCheck %s
+
+model = models.resnet18()
+model.training = False
+
+tensor = torch.randn(32,3,32,32)
+
+mb = torch_mlir.ModuleBuilder()
+
+with mb.capture_function("res18", [tensor]) as f:
+  result = model(tensor)
+  f.returns([result])
+
+print(mb.module)
+
+# for now we just check the output shape
```

Comment:
is there really much value in checking this in right now as a failing test?

Can you do like in frontends/pytorch/test/acap_export/test_export_multi_out.py and actually check the torch.kernel_call ops?

Reply:
There is for me: I've associated these with bugs that need to be fixed and would like to flip them to green as I make progress. Right now it is failing while building the function, so there is nothing to check yet.